### PR TITLE
handle aliases better

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
What names do you suggest then? :)

**Comment by EPashkin** [comment]
Also `get_chunk` that actually  adds chunk IMHO bad name.

**Comment by EPashkin** [comment]
`get_type_id`->`is_fundamental`
and second is tough, maybe just `get`->`add` and comment about returning bool

**Comment by GuillaumeGomez** [comment]
Renamed and updated a bit.

**Comment by EPashkin** [comment]
For `add_chunk_for_type` called `env.type_` internally, then call `env.type_` again on result IMHO just need bool

**Comment by sdroege** [comment]
Seems fine to me but maybe this should be part of the `is_fundamental()` method on the types directly? It is lying currently for alias types :)

**Comment by GuillaumeGomez** [comment]
Well, it's an alias so it's lying that much.

**Comment by EPashkin** [comment]
Maybe instead first function better add `is_fundamental_type` to library?

**Comment by EPashkin** [comment]
and remove type returning from `add_chunk_for_type` and use `is_fundamental_type`, this IMHO make code cleaner

**Comment by GuillaumeGomez** [comment]
That will be better this way indeed. Let's go for that!

**Comment by GuillaumeGomez** [comment]
Updated. I decided to create a function in `library` directly in case we'll need it again in the future.

**Comment by sdroege** [comment]
Still good for me :)

File: src/library.rs [hunk 1]
```diff
@@ -779,6 +783,15 @@ impl Type {
         }
     }
 
+    /// If the type is an Alias containing a fundamental, it'll return true (whereas
+    /// `is_fundamental` won't).
+    pub fn is_fundamental_type(&self, env: &Env) -> bool {
```
Comment (EPashkin) [inline src/library.rs]: Seems old `is_fundamental` used only internally so we can rename it to `is_fundamental_internal" (maybe make not public)

Comment (EPashkin) [reply src/library.rs]: But IMHO it good as is.

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
All seems ok, waiting for apveyor just in case

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks again

**Pull request merged**